examples/funloc/analysis_fun.py

```diff
@@ -87,7 +87,8 @@
 params.proj_nums = [[1, 1, 0],  # ECG
                     [1, 1, 2],  # EOG
                     [0, 0, 0]]  # Continuous (from ERM)
-params.autoreject_thresholds = True  # Set to True to use Autoreject module to set global epoch rejection thresholds  # noqa
+params.autoreject_thresholds = True  # Set to True to use Autoreject module to set global epoch rejection thresholds
+params.autoreject_eog = False  # Set to True to use EOG Autoreject criterion to reject trials based on EOG
```

Comment:
I'd rather do `params.autoreject_types` that defaults to `('meg', 'eeg')` (and you can do `('meg', 'eeg', 'eog')` if you want)